client/src/Utils/sigmfMetadata.ts

```diff
@@ -117,13 +119,25 @@ export class SigMFMetadata {
   }
 
   getThumbnailUrl() {
-    return this.getFullFilePath() + '.jpeg';
```

Comment:
What the ger sullfilepath would return before ? wouldnt be easy to change the apy to just accept that call ?

Reply:
The url originally generated by this call is specific to the storage of the file required. Changes were made to allow for the api URL which protects the sas token, to be generated correctly.